raw_data/preprocessed/africa/liberia_english.txt

```diff
@@ -0,0 +1,376 @@
+
+Environmental Education
+(102) The Agency shall, in consultation with the Ministry of Education take appropriate measures for the integration in schools, colleges and university curricula of environment education.
+
+State of the Environment Report
+(103) [1] The Agency shall every 5 years publish the Report on the State of the Environment of Liberia; [2] The Report shall in addition to other matters as may be prescribed, specify the main activities of the Agency and Line Ministries regarding the protection of the environment; [3] The Report shall take into account the plans and activities of the County and District Committees established under this Law; [4] The Agency may, publish any other information it considers necessary for information and public education on the environment.
+
+Record Keeping
+(104) [1] The Agency may by published notice prescribe the activities for which records shall be kept by owners of premises and operators for the purposes of this Law, the contents of those records and the manner in which they shall be kept; [2] The records kept in accordance with subsection [1] or any other records kept, for purposes of this Law, at the site of an establishment of undertaking shall be made available to the Agency or an environment inspector for the purposes of: [a] an environmental audit; [b] environmental monitoring; [c] pollution control; [d] inspection; [e] any other purpose that may be prescribed by the Agency. [3] The records kept pursuant to this section and any other records kept for purposes of this Act or the regulation made under this Law may be copied and transmitted to the Agency may be determined; [4] The Agency shall keep all records transmitted to it as public records and any person may on the application to the Agency, be granted access to the said records on the payment of a minimal fee prescribed by the Agency.
+
+PART XII - OFFENCES 
+
+Offences Relating to Environmental Impact Assessment
+(105) Any person who: [a] fails to publish a notice of intent; in accordance with section 7; [b] fails to submit a project brief in accordance with section 8; [c] fails to conduct a scoping process in accordance with section 11; [d] fails to prepare an environmental impact statement contrary to section 13 and/or section 14; [e] fails to prepare an environmental mitigation and implementation strategy in accordance with section 15; [f] fraudulently makes a false statement on an environmental impact statement, study report or any other documents submitted in relation to the environment impact assessment requirement under this Law commits an offence and is liable on conviction, to imprisonment for a period not exceeding 10 years or to a fine not exceeding 25,000 US DOLLARS or to both.
```

Comment:
Please replace:

* `intent; in accordance` with `intent in accordance`
* `liable on conviction, to imprisonment` with `liable on conviction to imprisonment`